spring-components/rdf4j-spring-demo/src/main/java/org/eclipse/rdf4j/spring/demo/ArtDemoCli.java

```diff
@@ -0,0 +1,90 @@
+/*******************************************************************************
+ * Copyright (c) 2021 Eclipse RDF4J contributors, Aduna, and others.
```

Comment:
This is not the correct copyright header I'm afraid. It should be this:

```
/*******************************************************************************
 * Copyright (c) 2021 Eclipse RDF4J contributors.
 * All rights reserved. This program and the accompanying materials
 * are made available under the terms of the Eclipse Distribution License v1.0
 * which accompanies this distribution, and is available at
 * http://www.eclipse.org/org/documents/edl-v10.php.
 *******************************************************************************/
```

Note that there's no ", Aduna, and others." <- that is something that is _only_ used for our legacy code from the Sesame project, not for anything new.

See https://github.com/eclipse/rdf4j/blob/master/.github/CONTRIBUTING.md#code-formatting for details.

Reply:
Sorry for that! I just opened a random source file and got the header from there. Not a safe practice, it seems ;-)

Reply:
fixed.